docs/content/manual/_incoming/2528-check-logs-harvester.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Check logs on Harvester
```

Comment:
How are we planning to move this to regression in future? I'm not sure how does updating the of test cases work here.

Reply:
The general way we do it is that we add in the tests for the release then we tag it, then we move them afterwards to the regressions. So that you can check any tagged release for the new tests for that release. I just tagged the release.

Reply:
Got it. Thanks